pyonfx/convert.py

```diff
@@ -63,6 +95,8 @@ def time(ass_ms: Union[int, str]) -> Union[str, int, ValueError]:
         """
         # Milliseconds?
         if type(ass_ms) is int and ass_ms >= 0:
+            # From https://github.com/Aegisub/Aegisub/blob/6f546951b4f004da16ce19ba638bf3eedefb9f31/libaegisub/include/libaegisub/ass/time.h#L32
+            ass_ms = (ass_ms + 5) - (ass_ms + 5) % 10
```

Comment:
I can't understand why we need to reduce precision, especially in such a delicate method.

Reply:
It rounds the milliseconds to centiseconds but the ASS format only supports centiseconds, so it doesn't reduce the precision.

Why should we do it?
It helps to have better precision.
Example: 49 ms is closest to 50 ms, but currently, it would floor it to 40 ms.

Reply:
Got it, amazing job :)
I never noticed it.

We should just document this fact better, maybe by adding an extra line of comment

Reply:
Done